conda/cli/install.py

```diff
@@ -379,18 +464,57 @@ def install(args, parser, command="install"):
             except UnicodeError:
                 raise CondaError(
                     "Error reading file, file should be a text file containing"
-                    " packages \nconda create --help for details"
+                    " packages \nconda update --help for details"
                 )
         if "@EXPLICIT" in specs:
             # short circuit to installing explicit if explicit specs are provided
-            explicit(specs, prefix, verbose=not context.quiet)
-            return
+            raise CondaError(
+                "Cannot update packages from an explicit file. Please try"
+                "\n`conda install` instead"
+            )
     specs.extend(common.specs_from_args(args_packages, json=context.json))
```

Comment:
Looks like `conda update` does not support the plugin system. Is this intentional? I tried passing an environment.yml and it crashes with a not-so-informative message:

```
$ conda update --file tests/env/support/with-pip.yml 

CondaValueError: could not parse 'name: pip' in: tests/env/support/with-pip.yml
```

I think we should still support the plugin system in `conda update` but only allow name-only specs. That also excludes the `pip` section, I guess.